tests/deprecated_api/test_remove_1-5.py

```diff
@@ -30,104 +29,6 @@
 from tests.helpers.utils import no_warning_call
 
 
-def test_v1_5_0_old_callback_on_save_checkpoint(tmpdir):
-    class OldSignature(Callback):
-        def on_save_checkpoint(self, trainer, pl_module):
-            ...
-
-    model = BoringModel()
-    trainer_kwargs = {"default_root_dir": tmpdir, "checkpoint_callback": False, "max_epochs": 1}
-    filepath = tmpdir / "test.ckpt"
-
-    trainer = Trainer(**trainer_kwargs, callbacks=[OldSignature()])
-    trainer.fit(model)
-
-    with pytest.deprecated_call(match="old signature will be removed in v1.5"):
-        trainer.save_checkpoint(filepath)
-
-    class NewSignature(Callback):
-        def on_save_checkpoint(self, trainer, pl_module, checkpoint):
-            ...
-
-    class ValidSignature1(Callback):
-        def on_save_checkpoint(self, trainer, *args):
-            ...
-
-    class ValidSignature2(Callback):
-        def on_save_checkpoint(self, *args):
-            ...
-
-    trainer.callbacks = [NewSignature(), ValidSignature1(), ValidSignature2()]
-    with no_warning_call(DeprecationWarning):
-        trainer.save_checkpoint(filepath)
-
-
-class BaseSignatureOnLoadCheckpoint(Callback):
-    def __init__(self):
-        self.on_load_checkpoint_called = False
-
-
-class OldSignatureOnLoadCheckpoint(BaseSignatureOnLoadCheckpoint):
-    def on_save_checkpoint(self, *args) -> Dict[str, Any]:
-        return {"a": 0}
-
-    def on_load_checkpoint(self, callback_state) -> None:
-        assert callback_state == {"a": 0}
-        self.on_load_checkpoint_called = True
-
-
-class NewSignatureOnLoadCheckpoint(BaseSignatureOnLoadCheckpoint):
-    def on_save_checkpoint(self, trainer, pl_module, checkpoint) -> dict:
-        return {"something": "something"}
-
-    def on_load_checkpoint(self, trainer, pl_module, checkpoint):
-        assert checkpoint == {"something": "something"}
-        self.on_load_checkpoint_called = True
-
-
-class ValidSignature2OnLoadCheckpoint(BaseSignatureOnLoadCheckpoint):
-    def on_save_checkpoint(self, trainer, pl_module, checkpoint) -> dict:
-        return {"something": "something"}
-
-    def on_load_checkpoint(self, *args):
-        assert len(args) == 3
-        self.on_load_checkpoint_called = True
-
-
-def test_v1_5_0_old_callback_on_load_checkpoint(tmpdir):
-
-    model = BoringModel()
-    trainer_kwargs = {"default_root_dir": tmpdir, "max_steps": 1}
-    chk = ModelCheckpoint(save_last=True)
-    trainer = Trainer(**trainer_kwargs, callbacks=[OldSignatureOnLoadCheckpoint(), chk])
-    trainer.fit(model)
-
-    with pytest.deprecated_call(match="old signature will be removed in v1.5"):
-        trainer_kwargs["max_steps"] = 2
-        cb = OldSignatureOnLoadCheckpoint()
-        trainer = Trainer(**trainer_kwargs, callbacks=cb, resume_from_checkpoint=chk.last_model_path)
-        trainer.fit(model)
-        assert cb.on_load_checkpoint_called
-
-    class ValidSignature1(BaseSignatureOnLoadCheckpoint):
```

Comment:
Should we keep the valid signature verification and move this to callback ?

Reply:
What for? We don't do it for other hooks, and the deprecated calls were removed